lib/http/HttpClient_Curl.hpp

```diff
@@ -208,7 +208,7 @@ class CurlHttpOperation {
          * Note that this API takes a pointer to a 'long' while we use
          * curl_socket_t for sockets otherwise.
          */
-        res = curl_easy_getinfo(curl, CURLINFO_LASTSOCKET, &sockextr);
+        res = curl_easy_getinfo(curl, CURLINFO_ACTIVESOCKET, &sockextr);
```

Comment:
CURLINFO_ACTIVESOCKET was added in  7.45.0. Just to be on safer side, should we check for the libcurl version (probably using [LIBCURL_VERSION_NUM](https://github.com/curl/curl/blob/aee4ebe59161d0a5281743f96e7738ad97fe1cd4/include/curl/curlver.h#L62)), and use right one accordingly.

Reply:
Should I change that in the current PR or a follow-up? :)

Reply:
Add a TODO comment here if it will be included in the following up PR?